source/features/cross-deleted-pr-branches.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {wrap} from '../libs/dom-utils';
+
+function init(): void {
+	const lastBranchAction = select.last('.TimelineItem-body .user-select-contain > span:not(.base-ref)');
+	if (!lastBranchAction) {
+		return;
+	}
+
+	if (!lastBranchAction.closest('.TimelineItem-body')!.textContent!.includes(' deleted ')) {
+		return;
+	}
+
+	const deletedBranchName = lastBranchAction.textContent!.trim();
+	const repoRoot = (select('.head-ref a')! as HTMLAnchorElement).href.replace(/\/tree.*$/, '');
+
+	for (const element of select.all('.commit-ref')) {
+		const branchName = element.textContent!.trim();
+		if (branchName !== 'unknown repository' && branchName === deletedBranchName) {
+			for (const deletedBranchElement of select.all('span', element)) {
+				deletedBranchElement.title = 'Deleted';
+				deletedBranchElement.style.textDecoration = 'line-through';
+				// The head-ref already has an a element
+				if (element.classList.contains('head-ref')) {
+					const linkElement = select('a', element)!;
+					linkElement.href = repoRoot;
+					linkElement.style.textDecoration = 'line-through';
+					linkElement.classList.remove('no-underline');
+				} else {
+					wrap(element, <a href={repoRoot}/>);
+				}
+			}
```

Comment:
This loop looks awkward. If the intent is to apply `line-through`, then let's use some CSS, example:

```css
//I think
.commit-ref[title="Deleted"],
.commit-ref[title="Deleted"] * {
	text-decoration: line-through !important;
}
```

```suggestion
			element.title = 'Deleted';

			if (element.classList.contains('head-ref')) {
				select('a', element)!.href = repoRoot;
			} else {
				wrap(element, <a href={repoRoot}/>);
			}
```

Reply:
awesome!! I knew there was a better way to do this